Dialog/TilesetEditDialog.cpp

```diff
@@ -586,7 +586,20 @@ void TilesetEditDialog::SetSelectedTile8x8(unsigned short tileId, bool resetscro
         ui->graphicsView_TilesetAllTile8x8->horizontalScrollBar()->setValue(0);
     }
 
-    ui->label_Tile8x8_ID->setText("Selected Tile8x8 Id: " + QString::number(tileId));
+    int fgtilenum = tilesetEditParams->newTileset->GetfgGFXlen() / 32;
+    int bgtilenum = tilesetEditParams->newTileset->GetbgGFXlen() / 32;
+    if((SelectedTile8x8 > (0x40 + fgtilenum)) && (SelectedTile8x8 < (0x5FF - bgtilenum)))
```

Comment:
I think this is an off-by-one error. For example, if there were 0 BG tiles, then the check `SelectedTile8x8 < 0x5FF` would not allow you to select tile number 0x5FF. I think it should be 0x600